<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A CS123 Final</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="View" name="view" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="sideBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="Scenes">
       <property name="title">
        <string>Scenes</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="sceneTypeSimple">
          <property name="text">
           <string>Scene 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sceneTypeCornell">
          <property name="text">
           <string>Scene 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sceneTypeSpecial1">
          <property name="text">
           <string>Scene 3</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QWidget</extends>
   <header>view.h</header>
   <slots>
    <slot>setDepthOfField(int)</slot>
    <slot>setScene1()</slot>
    <slot>setScene2()</slot>
    <slot>setScene3()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sceneTypeCornell</sender>
   <signal>clicked()</signal>
   <receiver>view</receiver>
   <slot>setScene1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sceneTypeSimple</sender>
   <signal>clicked()</signal>
   <receiver>view</receiver>
   <slot>setScene2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sceneTypeSpecial1</sender>
   <signal>clicked()</signal>
   <receiver>view</receiver>
   <slot>setScene3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
